service/resource/flanneldv1/desired.go

```diff
@@ -75,9 +75,6 @@ func newDaemonSet(customObject v1alpha1.FlannelConfig, etcdCAFile, etcdCrtFile,
 						"cluster":  keyv2.ClusterID(customObject),
 						"customer": keyv2.ClusterCustomer(customObject),
 					},
-					Annotations: map[string]string{
-						"seccomp.security.alpha.kubernetes.io/pod": "unconfined",
-					},
```

Comment:
I have removed seccomp annotation as anyone could justify why we used. BTW in the [k8s docs](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#seccomp) confined is the default value and it means seccomp is not applied to containers, so I expect it has any impact